src/models/MeasureSchemaValidator.ts

```diff
@@ -20,7 +20,8 @@ export const MeasureSchemaValidator = Yup.object().shape({
     .matches(
       /^[A-Z][a-zA-Z0-9]*$/,
       "Measure Library name must start with an upper case letter, followed by alpha-numeric character(s) and must not contain spaces or other special characters."
-    ),
+    )
+    .max(64, "Measure Library name cannot be more than 64 characters."),
```

Comment:
As per ACs, message would be: **Library name cannot be more than 64 characters.**

Reply:
I don't think this was resolved

Reply:
This was corrected to what the AC says